modules/ROOT/pages/6.0-release-notes-ui-and-ux-elements-and-components.adoc

```diff
@@ -0,0 +1,263 @@
+
+
+[[dialog-labels-render-correcttly]]
+==== dialog labels now rendering correctly
+
+Previously, {productname} dialog labels and other UI elements were rendered with HTML markup.
+
+With this release, the `text` and `label` properties of these elements are now rendered as plain text, as expected.
+
+
+[[editor-selection-setring]]
+==== `editor.selection.setRng`
+
+Calls to `editor.selection.setRng` now update the insertion point bookmark when focus is returned to the editor.
+
+For the end-user, this change means content dragged from outside the {productname} editor into the editable area will be placed into the {productname} document where the insertion point appears as the drag is completed.
+
+This is the expected behavior.
+
+
+[[sub-menu-items-now-read-by-screen-readers]]
+==== sub-menu items now read by screen readers
+
+Previously, navigating through {productname} menus using the keyboard did not cause screen readers to read out menu items.
+
+This has been corrected with this release.
+
+Screen readers now read menu items as the {productname} menus are traversed, as expected.
+
+NOTE: This patch was contributed by community member, https://github.com/westonkd[Weston Dransfield].
+
+
+[[tables-now-positioned-using-margins-not-float]]
+==== Tables are now positioned using `margin`, not `float`
+
+{productname} previously used `float` to position tables to the left or right of the viewport (the working area available to the {productname} editor). This caused multiple layout and UX problems.
+
+As of {productname} 6.0, tables are positioned using `margin` rather than `float`. This prevents these layout and UX problems occurring.
+
+However, when upgrading to {productname} 6.0, there are two things to be aware of:
+
+First, {productname} 6.0 does not alter existing documents. Extant {productname} documents with tables that were aligned to either _left_ or _right_ will still have the `float` property assigned when opened and edited unless and until their *alignment* is altered.
+
+If an extant {productname} document with an aligned table is opened using {productname} 6.0 and the *alignment* of the table is re-set, {productname} 6.0 will remove the `float` property and use the `margin` property to set the *alignment*.
+
+Second, if the previous behavior is required, a custom `format` will need to be created.
+
+
+[[text-presentation-corrections-on-some-UI-elements]]
+==== Text presentation corrections on some UI elements
+
+The text on some buttons and menu items incorrectly presented in Title Case.
+
+The underlying strings have been corrected and these items now present in sentence case, as expected.
+
+
+[[removed-or-deprecated-ui-and-ux-elements-and-components]]
+=== Removed or deprecated UI and UX elements and components
+
+[[mobile-theme]]
+==== `mobile` theme
+
+The `mobile` theme was deprecated when integrated support for mobile devices was  added to {productname}.
+
+The theme has been removed.
```

Comment:
```suggestion
The `mobile` theme was deprecated when integrated support for mobile devices was  added to the `silver` theme.

The `mobile` theme has been removed.
```
Maybe? It's just that the mobile theme was part of TinyMCE so that part of the sentence didn't make much sense.